src/OpenSage.Game/Logic/Orders/OrderProcessor.cs

```diff
@@ -15,12 +16,24 @@ public OrderProcessor(Game game)
             _game = game;
         }
 
+        private static NLog.Logger logger = NLog.LogManager.GetCurrentClassLogger();
+
         public void Process(IEnumerable<Order> orders)
         {
+            
             foreach (var order in orders)
             {
                 var player = _game.Scene3D.Players[(int) order.PlayerIndex];
 
+                if(order.OrderType == OrderType.SetCameraPosition)
```

Comment:
You could probably compose the message outside of the if/else statement and then just print the string inside